beetmoverscript/templates/mini_artifact_manifest.yml

```diff
@@ -0,0 +1,603 @@
+metadata:
+  description: Maps {{ product }} artifacts to pretty names
+  name: Beet Mover Manifest
+  owner: release@mozilla.com
+s3_bucket_paths:
+  devedition_candidates: pub/devedition/candidates/{{ version }}-candidates/build{{ build_number }}/
+  devedition_candidates_repacks: pub/devedition/candidates/{{ version }}-candidates/build{{ build_number }}/
+  fennec_candidates_repacks: pub/mobile/candidates/{{ version }}-candidates/build{{ build_number }}/
+  fennec_nightly: pub/mobile/nightly/
+  fennec_nightly_repacks: pub/mobile/nightly/
+  firefox_candidates: pub/firefox/candidates/{{ version }}-candidates/build{{ build_number }}/
+  firefox_candidates_repacks: pub/firefox/candidates/{{ version }}-candidates/build{{ build_number }}/
+  firefox_nightly: pub/firefox/nightly/
+  firefox_nightly_repacks: pub/firefox/nightly/
+  thunderbird_candidates: pub/thunderbird/candidates/{{ version }}-candidates/build{{ build_number }}/
+  thunderbird_candidates_repacks: pub/thunderbird/candidates/{{ version }}-candidates/build{{ build_number }}/
+  thunderbird_nightly: pub/thunderbird/nightly/
+  thunderbird_nightly_repacks: pub/thunderbird/nightly/
+mapping:
+  win32:
+    partials[partial].artifact_name:
+      default: partials[partial].artifact_name
+      devedition:
+        candidates:
+          pretty_name: update/{{ platform }}/{{ locale }}/firefox-{{ partials[partial].previousVersion }}-{{ version }}.partial.mar
+          pretty_path:
+          - update/{{ platform }}/{{ locale }}/firefox-{{ partials[partial].previousVersion }}-{{ version }}.partial.mar
```

Comment:
I know I made some of these suggestions initially, but now that it's populated, it looks like the pretty name and pretty path contain the exact same info, which leads me to the questions/observations:

1. do we gain anything by having a `pretty_name` defined if it's duplicated in `pretty_path`?
2. if we do gain something, should the `pretty_name` be the filename without any path info?

I'm also torn whether we'll ever use multiple `pretty_path`s (YAGNI) or if we'll be glad we have support for it for certain files. I bet we'll gather more opinions about this when we send this out more widely.

Reply:
pretty_name = s3_key
pretty_path = destinations
... in the current yml templates.
I can change it back to "s3_key" and "destinations" if you want!

Reply:
Mihai would know best but I think you're right Aki, that we could do without pretty_name/s3_key if it's just the basename of pretty_path/destinations